.github/workflows/build.yml

```diff
@@ -0,0 +1,74 @@
+name: Python package
+
+on: [pull_request]
+
+env:
+  INSTALL_DIR: ${{ github.workspace }}/install
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+        os: [ubuntu-latest]
+        include:
+        - python-version: 3.6
+          os: macos-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      env:
+        LD_LIBRARY_PATH: "${{ env.INSTALL_DIR }}/lib:${{ env.INSTALL_DIR }}/lib64"
```

Comment:
Can we try `sudo make install` instead of dealing with this path garbage?

Reply:
But we would still need to set the PYTHONPATH right?

Reply:
I will give it a go!

Reply:
It *might* work, so why not. You might need to do `-DCMAKE_INSTALL_PREFIX=/usr`. But I don't think it's that important to warrant spending a lot of time on.

Reply:
`sudo make install` did not play nice with MacOS, so will leave it like it is for now :disappointed: